docs/conf.py

```diff
@@ -70,7 +70,7 @@
 autodoc_default_flags = ["members", "inherited-members"]
 
 intersphinx_mapping = {
-    "micropip": (f"https://micropip.pyodide.org/en/v{micropip.__version__}/", None)
+    "micropip": (f"https://micropip.pyodide.org/en/v{micropip.__version__}/", None)  # type: ignore[attr-defined]
```

Comment:
This creates a mypy-tests error, presumably it fixes an error in plain mypy... Maybe we could add an exclude pattern to mypy-tests to skip this file?

Reply:
I guess my local micropip version was a problem.